### PR TITLE
added 7 api for enhancing (tor) The One Ring

### PR DESCRIPTION
**Pull request by Aragent**

### CODE REVIEW

**Review by Reonin — CHANGES_REQUESTED** [review]
Hi @Aragent !
The versioning folders for tor-dice-checker etc should be the number referenced in the respective script.json. so the folder for those should be 
'tor-dice-checker/**0.1.1**/README.md'

It would be nice if each new script we're broken up into it's own pull request for clarity and if one breaks after the fixes we don't need to pull back all of the scripts and only the ones that are broken. 

Otherwise these apis look great

**Comment by dave-roll20** [comment]
Please limit pull requests to one API script.

**Comment by shdwjk** [comment]
I talked to Nathanael Winget about this, it was a misunderstanding regarding policies of the repo.  He's getting it resolved.

File: tor-dice-checker/script.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-dice-checker.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-dice-checker/script.json:14]: JSON does not permit hanging `,` (comma) in arrays and objects.

File: tor-highlight-token-on-turn/script.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-highlight-token-on-turn.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-highlight-token-on-turn/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

File: tor-init-attributes/script.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-init-attributes.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-init-attributes/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

File: tor-mooks-state-checker/script.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-mooks-state-checker.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-mooks-state-checker/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

File: tor-stance-to-turn-order/script.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-stance-to-turn-order.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-stance-to-turn-order/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

File: tor-stances/script.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-stances.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-stances/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

File: tor-state-checker/script.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  "authors": "Michael Heilemann, Stefan Unterhuber, Stuart Johnson, David Domínguez, Eric Le Bourvellec, Michael (aka Aragent) I",
+  "roll20userid": "61774,1227702,383147,63716,211627,2624774",
+  "patreon": "",
+  "dependencies": [],
+  "modifies": {},
+  "conflicts": [],
+  "script": "tor-state-checker.js",
+  "useroptions": [],
+  "previousversions": [
+    "0.1.1",
```
Comment (shdwjk) [inline tor-state-checker/script.json:14]: JSON does not permit hanging , (comma) in arrays and objects.

**Review by shdwjk — CHANGES_REQUESTED** [review]
They won't be able to merge your changes until you fix your `script.json` files.  They all contain:
```
"previousversions": [
  "0.1.1",
]
```
which is invalid JSON.  You need to use:
```
"previousversions": [
  "0.1.1"
]
```
(no hanging comma).  I highly recommend using a json linter like [jsonlint.com](http://jsonlint.com).

**Comment by NicholasR77** [comment]
@Aragent This pull request has been open for quite a while with no changes. As such, I'm going to go ahead and close it. Please feel to re-submit or re-open whenever you're ready for us to review the requested changes here.

**Pull request closed without merging**